test/mocks/event/mocks.h

```diff
@@ -175,6 +176,34 @@ class MockTimer : public Timer {
   bool* timer_destroyed_{};
 };
 
+class MockScaledRangeTimerManager : public ScaledRangeTimerManager {
+public:
+  RangeTimerPtr createTimer(TimerCb callback) override {
+    return RangeTimerPtr{createTimer_(std::move(callback))};
+  }
+  MOCK_METHOD(RangeTimer*, createTimer_, (TimerCb));
+  MOCK_METHOD(void, setScaleFactor, (double), (override));
+};
+
+class MockRangeTimer : public RangeTimer {
+public:
+  MockRangeTimer() {}
+  MockRangeTimer(MockScaledRangeTimerManager* manager);
+
+  // Timer
+  MOCK_METHOD(void, disableTimer, (), (override));
+  MOCK_METHOD(void, enableTimer,
+              (std::chrono::milliseconds, std::chrono::milliseconds,
+               const ScopeTrackedObject* scope),
+              (override));
+  MOCK_METHOD(bool, enabled, (), (override));
+
+  const ScopeTrackedObject* scope_{};
+  bool enabled_{};
```

Comment:
scope_ and enabled_ seem unused.

Reply:
Yep, overzealous copy-paste. I'll remove them in the next commit.